doc/api/cli.md

```diff
@@ -443,6 +443,63 @@ Affects the default output directory of:
 * [`--heap-prof-dir`][]
 * [`--redirect-warnings`][]
 
+### `--disable-warnings=code-or-type`
```

Comment:
Can this flag be specified multiple times? Or possibly multiple codes be specified in a comma separated list?

Reply:
Multiple times (see the tests).

Reply:
Multiple times only. I did not add support for comma separated (unless that is handled by the general args parser)

Reply:
Can you add a test that uses a comma separated list, so we know how it would behave in that case?

Reply:
If the idea is for the flag to be used once per warning code, then `--disable-warning=` (not plural) would make the most sense on the name.

Reply:
It can be used for both individual warnings codes, and warning types.

Reply:
`--disable-warning` is winning.

https://twitter.com/ArrowoodTech/status/1726670254070513879

I will adjust this soon 👍